### DISCUSSION TITLE
Jak zmodyfikować avability set w Azure?

### DISCUSSION THREAD

**Discussion by tekch** [AzureCommunityPL/group#126] [Pomoc] [answered]
Chciałbym zamienic dwa serwery w avability set na dwa inne, a nie uczyniłem tego przy tworzeniu serwerów z appliance. Jakie rozwiązanie polecacie.

**Comment by wi5nia** [comment, marked as answer]
Swtorzyc dwa nowy serwery i dodac podczas tworzenia do tego AS, pozniej dwa stare usunac
[Dokumentacja](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/change-availability-set)

**Reply by tekch** [reply]
Tego się obawiałem, ze raczej tylko przy tworzeniu VM dało się to tyko zrobić.. Dziękuję.
